### PR TITLE
Fix Div and DivExp holding a high gate on the CLK output when reset

### PR DESCRIPTION
**Pull request by nickfeisst**
This fixes an issue in the Div and DivExp modules where they would output a high gate on the clock (CLK) output after they are reset and the master clock module is not running, or the phase input was not changing. The fix waits for the phase input to change before the clock output of the Div module is allowed to output a high gate output.

### CODE REVIEW

**Comment by zezic** [comment]
@nickfeisst Thanks for your PR! Will merge right after you adjust it according to the comments above.

**Comment by nickfeisst** [comment]
@zezic I'm not quite sure that I understand what you mean about the comments above. The only comment I see is mine, and it is a description of what is contained within the changes of the Pull Request that I have submitted. :)

**Review by zezic — CHANGES_REQUESTED** [review]

**Comment by zezic** [comment]
@nickfeisst Sorry. I forgot to publish my review comments! Here they are.

**Comment by nickfeisst** [comment]
That should fix those issues, I hope. I'm a little new to using Github, so I'm not entirely sure what I'm doing. 😂

**Review by zezic — APPROVED** [review]

**Comment by zezic** [comment]
Perfect! Thanks. I will test it with different cases tomorrow and release a new plugin version. It looks like it should work properly, but anyway, I have to test it with different cases before the release (just in case).

**Comment by nickfeisst** [comment]
Thanks! I really appreciate these modules - your Clock and Div modules are in almost every VCV Rack patch I've created.

**Pull request merged**
